src/find/mod.rs

```diff
@@ -104,6 +104,7 @@ fn parse_args(args: &[&str]) -> Result<ParsedInfo, Box<dyn Error>> {
 }
 
 #[allow(clippy::borrowed_box)] // FIXME?
+#[allow(clippy::unnecessary_wraps)] // FIXME?
```

Comment:
Why did you add this?

Reply:
The ci failed with `clippy::unnecessary-wraps` error

Reply:
Please fix the issue into a separate PR
A PR should focus on one thing :)
thanks